### PR TITLE
Fix 1477: Bigquery Usage - queryConfig

### LINKED ISSUE #1477: Bigquery Usage breaks if queryConfig is not found.
The Code breaks when it doesn't find queryConfig fetched from bigquery-logging
> queryConfig = payload["jobChange"]["job"]["jobConfig"][
                            "queryConfig"
                        ]

### PR DESCRIPTION
**Pull request by ayush-shah**
### Describe your changes :

Fix #1477, added fix for the bug 
if queryConfig is not in the jobConfig logs, it will skip that log entry
One of the users in support faced this error
#
### Type of change :

- [x] Bug fix

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Ingestion: @harshach @pmbrull

### CODE REVIEW

File: ingestion/src/metadata/ingestion/source/bigquery_usage.py [hunk 1]
```diff
@@ -68,42 +68,51 @@ def next_record(self) -> Iterable[TableQuery]:
...
+            for entry in logger.list_entries():
+                timestamp = entry.timestamp.isoformat()
+                timestamp = datetime.strptime(timestamp[0:10], "%Y-%m-%d")
+                if timestamp >= start and timestamp <= end:
+                    if ("query" in str(entry.payload)) and type(
+                        entry.payload
+                    ) == collections.OrderedDict:
+                        payload = list(entry.payload.items())[-1][1]
+                        if "jobChange" in payload:
+                            print(f"\nEntries: {payload}")
```
Comment (harshach) [inline ingestion/src/metadata/ingestion/source/bigquery_usage.py:81]: can we make this into logger.debug

File: ingestion/src/metadata/ingestion/source/bigquery_usage.py [hunk 2]
```diff
@@ -68,42 +68,51 @@ def next_record(self) -> Iterable[TableQuery]:
...
+                            if (
+                                "queryConfig"
```
Comment (harshach) [inline ingestion/src/metadata/ingestion/source/bigquery_usage.py:83]: so this is where we have the queryText?

**Review by harshach — CHANGES_REQUESTED** [review]

File: ingestion/src/metadata/ingestion/source/bigquery_usage.py (hunk 2 above)
Comment (ayush-shah) [reply ingestion/src/metadata/ingestion/source/bigquery_usage.py:83]: yes, queryConfig has the queryText, but it's an optional response parameter, that's why the user was facing KeyError.

File: ingestion/src/metadata/ingestion/source/bigquery_usage.py (hunk 1 above)
Comment (ayush-shah) [reply ingestion/src/metadata/ingestion/source/bigquery_usage.py:81]: yes, on it.

**Review by harshach — APPROVED** [review]

**Pull request merged**